branch-config.mk

```diff
@@ -19,8 +19,17 @@
 # conflicts on multiple places.
 
 
-# Name of the development branch. This could be master, fXX-release, rhelX-branch, rhel-X ...
-GIT_BRANCH ?= master
+# Name of the expected current git branch.
+# (Expected because it might be in fact a different one by mistake.)
```

Comment:
What exactly do you mean by that?

Reply:
Oh. I guess it is redundant. Basically I wrote "expected" and then thought, expected by who, where, what?